rust/hyperlane-base/src/settings/parser/connection_parser.rs

```diff
@@ -94,6 +94,12 @@ pub fn build_cosmos_connection_conf(
         None
     };
 
+    let minimum_gas_price = chain
+        .chain(err)
+        .get_opt_key("minimumGasPrice")
```

Comment:
iiuc you renamed to gas_price on the cosmos ConnectionConf - looks like here & in a few other places "minimum" is still used